packages/angular-recipe-app/src/app/recipes/recipe-detail/recipe-detail.component.html

```diff
@@ -1,50 +1,51 @@
-<ng-template [ngIf]="selectedRecipe" [ngIfElse]="placeholderText">
-  <div class="row">
-    <div class="col-xs-12">
-      <img
-        class="img img-responsive"
-        [src]="selectedRecipe.imageUrl"
-        [alt]="selectedRecipe.name"
-      />
-    </div>
-  </div>
-  <div class="row">
-    <div class="col-xs-12">
-      <h1>{{ selectedRecipe.name }}</h1>
-    </div>
-  </div>
-  <div class="row">
-    <div class="col-xs-12">
-      <div class="btn-group" appDropdown>
-        <button type="button" class="btn btn-primary dropdown-toggle">
-          Recipe Actions <span class="caret"></span>
-        </button>
-        <ul class="dropdown-menu">
-          <li><a href="#" (click)="addIngredients()">Shopping List</a></li>
-          <li>
-            <a href="#">Edit Recipe</a>
-          </li>
-          <li>
-            <a href="#">Delete Recipe</a>
-          </li>
-        </ul>
-      </div>
-    </div>
+<div class="row">
+  <div class="col-xs-12">
+    <img
+      class="img img-responsive"
+      [src]="selectedRecipe.imageUrl"
+      [alt]="selectedRecipe.name"
+    />
   </div>
-  <div class="row">
-    <div class="col-xs-12">{{ selectedRecipe.description }}</div>
+</div>
+<div class="row">
+  <div class="col-xs-12">
+    <h1>{{ selectedRecipe.name }}</h1>
   </div>
-  <div class="row">
-    <div class="col-xs-12">
-      <ul class="list-group">
-        <li
-          class="list-group-item"
-          *ngFor="let ingredient of selectedRecipe.ingredients"
-        >
-          {{ ingredient.name }}: {{ ingredient.amount }}
+</div>
+<div class="row">
+  <div class="col-xs-12">
+    <div class="btn-group" appDropdown>
+      <button type="button" class="btn btn-primary dropdown-toggle">
+        Recipe Actions <span class="caret"></span>
+      </button>
+      <ul class="dropdown-menu">
+        <li>
+          <a style="cursor: pointer" (click)="addIngredients()"
```

Comment:
`style="cursor: pointer"` is a code smell and hard to maintain. `a` should have this style by default, could you explain why do you have to specify it as an inline style. If something overwrite it, please find an other option to patch it, for example with a global style.

Reply:
I completely agree with you, Max introduced these upon removing the empty href's and as I see it as a code smell, too, I stopped following his example. I'm going to ignore this for now and clean it up later as it is not very Angular-specific.